unsymlink-lib

```diff
@@ -193,7 +193,8 @@ class MigrationState(object):
                 # files contained within them
                 if details[0] == 'dir':
                     continue
-                f = os.path.join(self.eroot, f.encode(_encodings['fs']).lstrip(b'/'))
+                f = f.encode(_encodings['fs'])
```

Comment:
I think we need to split `self.eroot` into `ROOT` and `EPREFIX` here, and still prepend `ROOT` but I guess this is fine for the time being.

Reply:
Got an idea: read EPREFIX from portage.const and reconstruct ROOT. Let me try again.

Reply:
Make a split of eroot into root and eprefix.  What do you think?